internal/scm/bitbucketserver/bitbucket_server.go

```diff
@@ -318,44 +318,53 @@ func (b *BitbucketServer) GetPullRequests(ctx context.Context, branchName string
 
 	var prs []scm.PullRequest
 	for _, repo := range repositories {
-		pr, getPullRequestErr := b.getPullRequest(client, branchName, repo)
```

Comment:
@ryancurrah Could you please take a look if these lines in the the bitbucket server looks good? 🙏

Reply:
I will try to validate these changes by Friday.

Reply:
Testing now.

Reply:
Seem to work as intended.